include/rmm/mr/device/per_device_resource.hpp

```diff
@@ -81,7 +81,8 @@
  * @endcode
  */
 
-namespace rmm::mr {
+namespace RMM_EXPORT rmm {
```

Comment:
This now means that the existing `RMM_EXPORT` on `get_map` in this file could be removed, IIUC. However, I suspect we should keep it (see also #1653) so that if we ever decide to tighten up the export list we don't lose that correctness.